src/Entity/ExcludeModification.php

```diff
@@ -2,71 +2,75 @@
 
 namespace App\Entity;
 
+use ApiPlatform\Doctrine\Orm\Filter\SearchFilter;
+use ApiPlatform\Metadata\ApiFilter;
 use ApiPlatform\Metadata\ApiResource;
 use ApiPlatform\Metadata\Delete;
 use ApiPlatform\Metadata\Get;
 use ApiPlatform\Metadata\GetCollection;
 use ApiPlatform\Metadata\Post;
-use ApiPlatform\Metadata\Put;
 use App\Repository\ExcludeModificationRepository;
+use App\Validator\Constraints\ExcludedMember;
 use Doctrine\ORM\Mapping as ORM;
 
 #[ApiResource(
     operations: [
         new Get(),
         new GetCollection(),
         new Post(),
-        new Put(),
         new Delete(),
-    ]
+    ],
+    formats: ['json'],
+    collectDenormalizationErrors: true
 )]
+#[ApiFilter(filterClass: SearchFilter::class, properties: ['payment', 'member'])]
 #[ORM\Entity(repositoryClass: ExcludeModificationRepository::class)]
+#[ORM\UniqueConstraint(columns: ['payment_uid', 'member_uid'])]
 class ExcludeModification
 {
     #[ORM\Id]
     #[ORM\Column(length: 255, unique: true)]
```

Comment:
### 🤔 Question: Should we have here `#[Assert\NotBlank]` propery?

I sow it in `member` entity and think that all other entities should have the same